src/lib/services/project-health-service.ts

```diff
@@ -22,7 +25,9 @@ export default class ProjectHealthService {
 
     private projectService: ProjectService;
 
-    calculateHealthRating: (project: IProject) => Promise<number>;
+    calculateHealthRating: (project: Pick<IProject, 'id'>) => Promise<number>;
```

Comment:
this one could also be `private`, if it isn't public for testing

Reply:
it is called by the scheduler